src/Common/tests/System/PlatformDetection.cs

```diff
@@ -12,6 +12,8 @@ namespace System
 {
     public static class PlatformDetection
     {
+        public static bool IsFullFramework => RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
```

Comment:
I can't replace all of them though, because 2 hits are in the Common Path and those files are included individually on every project that needs them, so if I change those 2 hits to use `PlatformDetection` if any test needs any of those 2 files then they will need a dependency on `PlatformDetection` as well. I could fix it for the projects that hit this now. The problem is if someone adds a test that brings any of those 2 files into their test project they would possibly struggle finding why their project is not building because they will be missing the `PlatformDetection.cs` dependency even if they don't use it in their tests.